### ISSUE TITLE
DataGrid Header and Bootstrap 5 Dark Mode

### ISSUE THREAD

**Issue by stsrki** [Megabit/Blazorise#5712]
### Discussed in https://github.com/Megabit/Blazorise/discussions/5711

<div type='discussions-op-text'>

<sup>Originally posted by **Alexander-JM-0224** August 22, 2024</sup>
Hello,

I am currently working on the UI of the web application of my employer, and i am supposed to implement a switchable Dark Theme.

Since we use Bootstrap 5 (5.3.3) i have used the JS script in the Bootstrap docs to make switching our Bootstrap theme possible and this works wich is very nice.
There is no other styling in our application (Blazor WASM .NET8) for the DataGrids, but the background colors do not all switch correctly and the inspector seems to indicate there is background-color coming from the Blazorise CSS.
Important note, i have already tried to give the DataGrids their own CSS class just for the background color. Even with !important it does nothing for me. 
When i switch the color off in the inspector everything is perfect.

Problem DataGrid Header
![Header-pict](https://github.com/user-attachments/assets/239bf259-1659-4219-aeda-d0ab662c86f8)
Inspector
![Inspector](https://github.com/user-attachments/assets/90af4637-f772-4b11-843d-c9304bb55199)
Result when Inspector turns background-color off
![Header-pict-dark](https://github.com/user-attachments/assets/7c44a37f-6348-4010-95db-9551af250188)
Same for a colum
![Column-pict](https://github.com/user-attachments/assets/0fa9dc46-ff83-41ed-bcb9-75b7bfb9c34c)
And the inspector
![Column-inspector](https://github.com/user-attachments/assets/373a5a60-b892-4d11-9997-db5287f7549f)
This is how they declare the DataGrid i do not think the problem starts here but ill give it anyways.
![Declaration](https://github.com/user-attachments/assets/e7523097-b549-4927-8d72-73aebd900cad)

Tested in Chrome and Firefox with Windows 11

What could be the cause of this issue? Are we doing something wrong? Do you need more information from me? Let me know!

Thanks in advance!
Alexander
</div>

**Comment by David-Moreira** [comment]
Have you tried defining the background color by using the Row Styling settings? I think we've added support for the Fixed Header to consider that background color.

**Comment by Alexander-JM-0224** [comment]
> Have you tried defining the background color by using the Row Styling settings? I think we've added support for the Fixed Header to consider that background color.

No, if we use RowStyling this only changes the background color of the DataGrid Rows, not te header. And it gets overruled by our theme settings so the RowStyling background color is only visible if i turn off our "Dark Mode BG" in the inspector.

**Comment by Alexander-JM-0224** [comment]
![Image](https://github.com/user-attachments/assets/30f3be9c-d57e-4f4)f4-8d6b-4803276fb16

**Comment by Alexander-JM-0224** [comment]
![Image](https://github.com/user-attachments/assets/8fe9c2f9-d5cb-4f05-bd71-b0b2fec9933c)
